include/uapi/linux/bpf.h

```diff
@@ -1226,6 +1226,9 @@ enum {
 
 /* Create a map that is suitable to be an inner map with dynamic max entries */
 	BPF_F_INNER_MAP		= (1U << 12),
+
+/* Create an over writable BPF_RINGBUF */
+	BFP_F_RB_OVER_WRITABLE	= (1U << 13),
```

Comment:
The kernel seems to favor 'overwritable' in a single word, including in the documentation of the perf ring buffer:
```
$ git grep -ni overwritable
$ git grep -ni over.writable
```

```suggestion
/* Create an overwritable BPF_RINGBUF */
	BFP_F_RB_OVERWRITABLE	= (1U << 13),
```

Reply:
Done! Thank you!